js_modules/dagster-ui/packages/app-oss/src/NUX/CommunityNux.tsx

```diff
@@ -21,14 +21,18 @@ export const CommunityNux = () => {
     'communityNux',
     (data) => data,
   );
-  const {data, loading} = useQuery(GET_SHOULD_SHOW_NUX_QUERY);
+  const {data} = useSuspenseQuery(GET_SHOULD_SHOW_NUX_QUERY);
   const [dismissOnServer] = useMutation(SET_NUX_SEEN_MUTATION);
```

Comment:
Can these have type parameters? (Come to think of it, why don't the existing ones? 🤔)

Reply:
Because this is `app-oss` which doesn't generate the types :\